File: app/src/main/java/com/meghanil/ganpatiaarto/ViewPagerActivity.java
package com.meghanil.ganpatiaarto;

import android.os.Bundle;
import androidx.viewpager.widget.ViewPager;
import androidx.appcompat.app.AppCompatActivity;
import android.util.Log;

import com.meghanil.ganpatiaarto.adapter.AartiPagerAdapter;

public class ViewPagerActivity extends AppCompatActivity {
   AartiPagerAdapter aartiPagerAdapter;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.content_view_pager);
        int position = getIntent().getIntExtra(MainActivity.POSITION, 0);
        ViewPager pager = (ViewPager) findViewById(R.id.viewPager);
        pager.setOffscreenPageLimit(1);
        Log.d("ViewPagerActivity", "onCreate: "+position);
        pager.setAdapter(new AartiPagerAdapter(getSupportFragmentManager(),this));
       pager.setCurrentItem(position,true);
    }

}

File: app/src/main/java/com/meghanil/ganpatiaarto/ViewPagerActivity.kt
package com.meghanil.ganpatiaarto

import android.os.Bundle
import android.util.Log
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager.widget.ViewPager
import com.meghanil.ganpatiaarto.adapter.AartiPagerAdapter

class ViewPagerActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.content_view_pager)
        val position = intent.getIntExtra(Util.POSITION, 0)
        val pager = findViewById<View>(R.id.viewPager) as ViewPager
        pager.offscreenPageLimit = 1
        Log.d("ViewPagerActivity", "onCreate: $position")
        pager.adapter = AartiPagerAdapter(supportFragmentManager, this)
        pager.setCurrentItem(position, true)
    }
}